Userland/Libraries/LibWeb/Streams/ByteLengthQueuingStrategy.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023, Shannon Booth <shannon.ml.booth@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Forward.h>
+#include <LibJS/Forward.h>
+#include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/Forward.h>
+#include <LibWeb/Streams/QueuingStrategyInit.h>
+
+namespace Web::Streams {
+
+// https://streams.spec.whatwg.org/#bytelengthqueuingstrategy
+class ByteLengthQueuingStrategy final : public Bindings::PlatformObject {
+    WEB_PLATFORM_OBJECT(ByteLengthQueuingStrategy, Bindings::PlatformObject);
+
+public:
+    static WebIDL::ExceptionOr<JS::NonnullGCPtr<ByteLengthQueuingStrategy>> construct_impl(JS::Realm&, QueuingStrategyInit const&);
+
+    virtual ~ByteLengthQueuingStrategy() override;
+
+    // The highWaterMark getter steps are:
```

Comment:
Also here move this comment inside getter and add spec link (https://streams.spec.whatwg.org/#blqs-high-water-mark)